shared/js/background.js

```diff
@@ -77,10 +77,12 @@ function Background() {
     // only show post install page on install
     if (details.reason.match(/install/)) {
         settings.ready().then( () => {
-          if (!settings.getSetting('hasSeenPostInstall')) {
+          const domain = window.location ? window.location.href : ''
+          const postInstallUrl = 'https://www.duckduckgo.com/app'
```

Comment:
I'm not sure if the `www` will always be there. E.g. if I install the extension from https://duckduckgo.com/app.

Also, if you use `match` the string gets converted to a regex ([MDN link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match)). It might be a good idea to escape the dots?